wger/gym/templates/gym/partial_user_list.html

```diff
@@ -15,11 +15,29 @@
 </script>
 
 {% if not show_gym %}
+    <script>
+        function compareMembers() {
+            var selected = [];
+            var new_url = "";
+            $("input:checkbox[name=selected_member]:checked").each(function(){
+                selected.push($(this).val());
+            });
+            for (var i=0; i < selected.length; i++) {
```

Comment:
Try replacing `for-loop & nested if-else` with `reduce` function i.e
```
new_url = selected.reduce(function(current,next){
    return current+'-or-'+next
});
```
Shorter, faster and more efficient.